doq/_shtab.py

```diff
@@ -0,0 +1,20 @@
+from argparse import Action
+
+FILES = DIRS = {}
```

Comment:
I think this should be FILE and DIR
https://github.com/iterative/shtab/blob/a81c26121c8148c886e8cf7d06d377dcf2062371/shtab/__init__.py#L43-L44

```
$ PYTHONPATH=/opt/tmp/test/lib/python3.11/site-packages /opt/tmp/test/bin/doq --help
Traceback (most recent call last):
  File "/opt/tmp/test/bin/doq", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/opt/tmp/test/lib/python3.11/site-packages/doq/cli.py", line 366, in main
    args = parse_options()
           ^^^^^^^^^^^^^^^
  File "/opt/tmp/test/lib/python3.11/site-packages/doq/cli.py", line 264, in parse_options
    ).complete = shtab.FILE
                 ^^^^^^^^^^
AttributeError: module 'doq._shtab' has no attribute 'FILE'. Did you mean: 'FILES'?
```